config/src/config/state_sync_config.rs

```diff
@@ -153,3 +156,19 @@ impl Default for DataStreamingServiceConfig {
         }
     }
 }
+
+#[derive(Copy, Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default)]
+pub struct DiemDataClientConfig {
+    pub timeout: Duration, // Timeout used by the network client when sending requests
+    pub data_summary_poll_interval: Duration, // Interval between data summary polls
+}
+
+impl Default for DiemDataClientConfig {
+    fn default() -> Self {
+        Self {
+            timeout: Duration::from_millis(10_000),
```

Comment:
Let's maybe reduce the default value. I mentioned this to Philip a while back, but I feel like 10 seconds is huge. Maybe 3 seconds?

Reply:
Updated `response_timeout` to smaller value (3 seconds). (I'm fine with any values between 1 and 10 secs. I did google for the common timeout values used for the wide area. It seems that any values around this range is reasonable. Anyway we can always change this timeout later.)